src/types/Todo.ts

```diff
@@ -0,0 +1,6 @@
+export interface Todo {
+  id?: number;
```

Comment:
`id` is required, you can create a new type without it for create todo functions
```typescript
type newTodo = Omit<Todo, 'id'>;
```
https://www.typescriptlang.org/docs/handbook/utility-types.html#omittype-keys